### ISSUE TITLE
Generate correct location path when "Use bnd workspace location" is unchecked

### ISSUE THREAD

**Issue by paulvi** [bndtools/bndtools#1337]
NullPointerException message when not using bnd workspace location

BndTools 3.1

![bndtools nullpointerexception when not using bnd workspace location](https://cloud.githubusercontent.com/assets/1614482/14376828/7c3e0986-fd9e-11e5-96d4-f323d80c7a6c.png)

 Please generate correct location path (derived from project name) when checkbox "Use bnd workspace location" is unchecked by user

**Comment by paulvi** [comment]
You may close this as the error happen only when pasting project name into location field. 
(That is Eclipse platform behavior)

There were some others strange thing the first time I tried to make a project outside of bdn workspace, but I cannot reproduce to make a screenshot.

Anyways, I bet my hat you are not checking how bndtools-Eclipse works for non-Workspace projects.

**Comment by bjhargrave** [comment]
Bndtools only support Bnd workspace.

**Comment by paulvi** [comment]
- [ ] Well, than this should be said on http://bndtools.org/ or at least on the Dialog above, like

and possibly checkbox should be renamed "Use this bnd workspace location",  
so that user knows that he/she need a folder under bnd workspace 

```
- [ ] Use this bnd workspace location
Only one bnd workspace in Eclipse workspace is supported.
```

**Comment by pkriens** [comment]
Fixed in cdb012012edecc616afa00250af3a351c16324b3
